### PR TITLE
Feature/#1 Jekyll site

### LINKED ISSUE #1: Setup Github Pages site

### PR DESCRIPTION
**Pull request by markspanbroek**
This closes #1
Added empty Jekyll site for http://sustainablesource.io.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'github-pages', group: :jekyll_plugins
```
Comment (svdo) [inline Gemfile]: Or did you mean this (cf https://jekyllrb.com/docs/plugins/):

```
group :jekyll_plugins do
    gem 'github-pages'
end
```

Comment (svdo) [inline Gemfile]: By the way, if and when needed, this is also cool: https://chumper.github.io/2016/01/04/blog-setup.html.

Comment (markspanbroek) [inline Gemfile]: They mean the same thing, ```:key => value``` can be abbreviated as ``key: value`` in Ruby.
By the way, I copied this line from https://github.com/github/pages-gem.

Comment (markspanbroek) [inline Gemfile]: PlantUML in Markdown, nice!

**Pull request merged**